maps/AzaleaTown.asm

```diff
@@ -14,23 +14,23 @@
 
 AzaleaTown_MapScripts:
 	def_scene_scripts
-	scene_script .DummyScene0, SCENE_AZALEATOWN_NOOP
-	scene_script .DummyScene1, SCENE_AZALEATOWN_RIVAL_BATTLE
-	scene_script .DummyScene2, SCENE_AZALEATOWN_KURT_RETURNS_GS_BALL
+	scene_script AzaleaTownNoopScene,              SCENE_AZALEATOWN_NOOP
+	scene_script AzaleaTownRivalBattleScene,       SCENE_AZALEATOWN_RIVAL_BATTLE
```

Comment:
So, in Gen 2, there are a lot of cases like these where a meaningful `SCENE` constant is only needed for a `coord_event`, but has a no-op `scene_script` anyway. I think it would make more sense to give them generic labels that indicate this, since things like "`AzaleaTownRivalBattleScene`" sound like they do something for the rival battle. it could just be `AzaleaTownNoopScene1`, `AzaleaTownNoopScene2`, etc.

Reply:
I see what you mean. Will do.